### DISCUSSION TITLE
В документации 2 версии ссылка недействительна

### DISCUSSION THREAD

**Discussion by Vladislav-Chistyakov** [yandex-maps-unofficial/vue-yandex-maps#353] [Q&A (2.x)] [answered]
По [ссылке](https://yandex-maps-unofficial.github.io/vue-yandex-maps/components/modules/controls/zoom.html#%D0%BF%D1%80%D0%B8%D0%BC%D0%B5%D1%80%D1%8B-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F) 
есть недействительная ссылка в линке - "Примеры использования"
![image](https://github.com/user-attachments/assets/58fc19ab-a559-4ca2-a5f7-4338dbf4bc8a)

Либо пока не реализовано, либо ссылка идёт не туда))

**Comment by Vladislav-Chistyakov** [comment]
Вообще искал, можно ли запретить пользователю плиближать при помощи скролла, реализовано ли это? (Знаю, что в яндекс картах есть такая возможность)

**Reply by daniluk4000** [reply]
Да, конечно

https://yandex-maps-unofficial.github.io/vue-yandex-maps/examples/map/behaviors.html

**Reply by Vladislav-Chistyakov** [reply]
Супер, спасибо!

**Comment by daniluk4000** [comment, marked as answer]
Реализовано, поправлю, спасибо)

**Reply by daniluk4000** [reply]
А это просто навигация. Страница маленькая, поэтому она никуда не ведет. А на элементы управления ссылка рабочая
